destination_sdk_v2.proto

```diff
@@ -0,0 +1,181 @@
+syntax = "proto3";
+option optimize_for = SPEED;
+option java_multiple_files = true;
+option go_package = "fivetran.com/fivetran_sdk/v2";
+package fivetran_sdk.v2;
+
+import "google/protobuf/timestamp.proto";
+import "common_v2.proto";
+
+// Fivetran (grpc client) <> DestinationConnector (grpc server)
+service DestinationConnector {
+    rpc ConfigurationForm (ConfigurationFormRequest) returns (ConfigurationFormResponse) {}
+    rpc Capabilities(CapabilitiesRequest) returns (CapabilitiesResponse) {}
+    rpc Test (TestRequest) returns (TestResponse) {}
+    rpc DescribeTable (DescribeTableRequest) returns (DescribeTableResponse) {}
+    rpc CreateTable(CreateTableRequest) returns (CreateTableResponse) {}
+    rpc AlterTable(AlterTableRequest) returns (AlterTableResponse) {}
+    rpc Truncate(TruncateRequest) returns (TruncateResponse) {}
+    rpc WriteBatch (WriteBatchRequest) returns (WriteBatchResponse) {}
+}
+
+message CapabilitiesRequest {}
+
+message CapabilitiesResponse {
+    BatchFileFormat batch_file_format = 1;
+    repeated DataTypeMappingEntry data_type_mappings = 2;
+    bool supports_history_mode = 3;
+}
+
+message DataTypeMappingEntry {
+    DataType fivetran_type = 1;
+    oneof destination_type {
+        DestinationType map_to = 2;
+        bool unsupported = 3;
+    }
+}
+
+message DestinationType {
+    string name = 1;
+    oneof mapping {
+        bool keep_same = 2;
+        DataType map_to = 3;
+    }
+    optional MaxValue max_value = 4;
+}
+
+message MaxValue {
+    oneof max_param {
+        int32 numeric_param = 1;
+        DecimalParams decimal_param = 2;
+        google.protobuf.Timestamp date_param = 3;
+    }
+}
+
+message DescribeTableRequest {
+    map<string, string> configuration = 1;
+    string schema_name = 2;
+    string table_name = 3;
+}
+
+message DescribeTableResponse {
+    oneof response {
+        bool not_found = 1;
+        string failure = 2;
+        Table table = 3;
+    }
+}
+
+message CreateTableRequest {
+    map<string, string> configuration = 1;
+    string schema_name = 2;
+    Table table = 3;
+}
+
+message CreateTableResponse {
+    oneof response {
+        bool success = 1;
+        string failure = 2;
+    }
+}
+
+message AlterTableRequest {
+    map<string, string> configuration = 1;
+    string schema_name = 2;
+    string table_name = 3;
+    repeated SchemaDiff changes = 4;
+}
+
+message SchemaDiff {
+    oneof change {
+        Column add_column = 1;
+        ChangeType change_column_type = 2;
+        PrimaryKey updated_primary_keys = 3;
```

Comment:
This field should be named `update_primary_keys` (without the `d`) for consistency. Let's fix it in the `engineering` repo too.